src/utils/util.js

```diff
@@ -139,14 +140,43 @@ function equalUrlsIgnoringSubpath(url1, url2) {
   return url1.origin.toLowerCase() === url2.origin.toLowerCase();
 }
 
+const dispatchNotification = async (title, body, channel, teamId, silent) => {
```

Comment:
instead of having the channel and team id, why not pass a callback for the `onclick` function? that way you don't have to ignore the parameters and it's a bit more malleable for future usage.

Reply:
I checked and it seems that we do need to pass the channel and teamId, because of 
```
ipcRenderer.on('notification-clicked', (event, {channel, teamId}) => {
  window.postMessage(
    {
      type: 'notification-clicked',
      message: {
        channel,
        teamId,
      },
    },
    window.location.origin
  );
});
```
we will need to go through the whole webapp and will need to refactor all notifications to rely on the custom onClick, instead of the teamId and channel. Maybe in a separate ticket?